pkg/cmd/roachtest/spec/cluster_spec.go

```diff
@@ -214,7 +214,7 @@ func awsMachineSupportsSSD(machineType string) bool {
 }
 
 func getAWSOpts(
-	machineType string, zones []string, volumeSize, ebsThroughput int, localSSD bool,
+	machineType string, zones []string, volumeSize, ebsThroughput int, localSSD bool, useSpotVMs bool,
```

Comment:
Give that we are passing too many aguements, it will be better to create a struct.